samples/sensor/bme68x/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _bme68x:
+
+Gas Sensor BME688
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample application sets up the BME68X gas sensor with the Bosch Sensor Environmental Cluster (BSEC) library.
+
+Requirements
+************
+
+Using this driver requires manually downloading the BSEC library. See :ref:`lib_bme68x` for more details.
+The sample supports the following devices:
+
+.. table-from-sample-yaml::
+
+Building and running
+********************
+
+This project outputs sensor data to the console.
+It requires a BME68X sensor.
+Board overlays for Thingy:91 and Thingy:53 are included.
```

Comment:
I only see board overlay files for Thingy:53, but I don't think this sentence is needed. @divipillai ?

Reply:
 If this is included to show the boards that support the sample, we could remove this sentence since it is mentioned in the requirement section, or is there any reason to mention the board overlays here?